faq/pt/proofreading_guidelines.php

```diff
@@ -8,7 +8,7 @@
 include_once($relPath.'pg.inc');
 include_once($relPath.'theme.inc');
 
-#maybe_redirect_to_external_faq("pt");
+maybe_redirect_to_external_faq("pt");
```

Comment:
this is a functional change, no?

Reply:
But will only affect the Portuguese Proofreading Guidelines. The other languages already have active redirects to the wiki.

Reply:
Yes, this is a functional change (so my statement about there not being one is incorrect). This is intentional -- I don't know why was commented out prior -- I assume it was a mistake somewhere.